Mail/Components/UnavailableMailboxListView.swift

```diff
@@ -31,8 +31,7 @@ struct UnavailableMailboxListView: View {
             return mailboxInfosManager.realmConfiguration
         }(),
         where: { mailbox in
-            @InjectService var accountManager: AccountManager
-            return mailbox.userId == accountManager.currentUserId && mailbox.isPasswordValid == false
+            isCurrentUserMailbox(mailbox) && mailbox.isPasswordValid == false && !isMailboxConsideredLocked(mailbox)
```

Comment:
I'm not a fan of relatively complex one-liners like this one. Please consider putting it behind a named func.